service/controller/app/v1/key/key.go

```diff
@@ -13,6 +13,18 @@ func AppName(customObject v1alpha1.App) string {
 	return customObject.Spec.Name
 }
 
+func CatalogName(customObject v1alpha1.App) string {
```

Comment:
```suggestion
func CatalogName(customResource v1alpha1.App) string {
```

We should use the new naming here.